demo/README.md

```diff
@@ -0,0 +1,203 @@
+# in-toto demo
+
+In this demo, we will use in-toto to secure a supply chain with a very simple
+workflow.
+Alice will be the project owner - she creates and signs the supply chain layout
+with her private key - and Bob and Carl will be the project functionaries - they
+carry out the steps of the supply chain as defined in the layout.
```

Comment:
Can you include a few more sentences to explain what is meant by "supply chain?" (note: consider using "software supply chain")  Likewise, it would help the introduction if "layout" were defined.  What does the layout for this simple workflow resemble?

You will likely cover in-toto's design in a presentation, but it would be beneficial to the reader to have a README of sorts for this.  The current [README](https://github.com/in-toto/in-toto/blob/master/README.md) doesn't include an overview of in-toto.  Perhaps you could link the specification here?